addons/knowledge/static/src/scss/knowledge_views.scss

```diff
@@ -25,6 +25,7 @@
         visibility: hidden;
         opacity: 0;
         transition: visibility 0.4s, opacity 0.4s linear;
+        color: grey !important;
```

Comment:
:grimacing:

Reply:
Yes this is not very pretty. Do you have any ideas on how to make it better?